kf bind-service panics

<!-- If you need to report a security issue with kf, send an email to kf-security@google.com. -->

## What version of kf?
07affc90f5622ae84a8d095c19e498cc74410d4c

<!-- Paste the output of kf version -->

## Expected Behavior

<!-- Briefly describe what you expect to happen -->

```
$ kf bind-service my-app my-big-table -c '{"role":"bigtable.user"}'
```

doesn't panic

## Actual Behavior

<!-- Briefly describe what is actually happening -->

```
$ kf bind-service my-app my-big-table -c '{"role":"bigtable.user"}'
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x370 pc=0x2073eda]

goroutine 1 [running]:
github.com/google/kf/pkg/kf/apps.BindService(0x0, 0xc0004a60c0)
	/Users/poy/workspace/kf-stuff/kf-master/pkg/kf/apps/util.go:21 +0x3a
github.com/google/kf/pkg/kf/apps.(*appsClient).BindService.func1(0xc0004d9b00, 0xc000719370, 0x7)
	/Users/poy/workspace/kf-stuff/kf-master/pkg/kf/apps/client.go:99 +0x36
github.com/google/kf/pkg/kf/apps.(*coreClient).Transform(0xc0006a8cd0, 0xc000719370, 0x7, 0x7ffeefbff998, 0x6, 0xc00023a560, 0x40, 0x234b200, 0x2256a01)
	/Users/poy/workspace/kf-stuff/kf-master/pkg/kf/apps/zz_generated.client.go:203 +0x9f
github.com/google/kf/pkg/kf/apps.(*appsClient).BindService(0xc0006a8cc0, 0xc000719370, 0x7, 0x7ffeefbff998, 0x6, 0xc0004a60c0, 0x0, 0xc000407d08, 0x1f212f4)
	/Users/poy/workspace/kf-stuff/kf-master/pkg/kf/apps/client.go:98 +0xc0
github.com/google/kf/pkg/kf/commands/service-bindings.NewBindServiceCommand.func1(0xc000680000, 0xc000107dc0, 0x2, 0x4, 0x0, 0x0)
	/Users/poy/workspace/kf-stuff/kf-master/pkg/kf/commands/service-bindings/bind-service.go:65 +0x214
github.com/spf13/cobra.(*Command).execute(0xc000680000, 0xc000107d40, 0x4, 0x4, 0xc000680000, 0xc000107d40)
	/Users/poy/workspace/go/pkg/mod/github.com/spf13/cobra@v0.0.5/command.go:826 +0x465
github.com/spf13/cobra.(*Command).ExecuteC(0xc00003e500, 0x20ddab4, 0xc000539f88, 0xc0000bc058)
	/Users/poy/workspace/go/pkg/mod/github.com/spf13/cobra@v0.0.5/command.go:914 +0x2fc
github.com/spf13/cobra.(*Command).Execute(...)
	/Users/poy/workspace/go/pkg/mod/github.com/spf13/cobra@v0.0.5/command.go:864
main.main()
	/Users/poy/workspace/kf-stuff/kf-master/cmd/kf/main.go:24 +0x28
```

## Steps to Reproduce the Problem

<!-- How can a maintainer reproduce this issue (be detailed) -->

--- a/pkg/kf/apps/client.go
+++ b/pkg/kf/apps/client.go
@@ -72,8 +72,8 @@ func (ac *appsClient) DeleteInForeground(namespace string, name string) error {
 // Restart causes the controller to create a new revision for the knative
 // service.
 func (ac *appsClient) Restart(namespace, name string) error {
-	_, err := ac.coreClient.Transform(namespace, name, func(a *v1alpha1.App) error {
-		a.Spec.Template.UpdateRequests++
+	_, err := ac.coreClient.Transform(namespace, name, func(app *v1alpha1.App) error {
+		app.Spec.Template.UpdateRequests++
 		return nil
 	})
 
@@ -95,7 +95,7 @@ func (ac *appsClient) Restage(namespace, name string) (app *v1alpha1.App, err er
 
 // BindService adds the given service binding to the app.
 func (ac *appsClient) BindService(namespace, name string, binding *v1alpha1.AppSpecServiceBinding) (app *v1alpha1.App, err error) {
-	return ac.coreClient.Transform(namespace, name, func(a *v1alpha1.App) error {
+	return ac.coreClient.Transform(namespace, name, func(app *v1alpha1.App) error {
 		BindService(app, binding)
 		return nil
 	})
@@ -103,7 +103,7 @@ func (ac *appsClient) BindService(namespace, name string, binding *v1alpha1.AppS
 
 // UnbindService removes the given service binding from the app.
 func (ac *appsClient) UnbindService(namespace, name, bindingName string) (app *v1alpha1.App, err error) {
-	return ac.coreClient.Transform(namespace, name, func(a *v1alpha1.App) error {
+	return ac.coreClient.Transform(namespace, name, func(app *v1alpha1.App) error {
 		UnbindService(app, bindingName)
 		return nil
 	})

